pages/blog/css-with-react.mdx

```diff
@@ -0,0 +1,274 @@
+  ]}
+/>
+
+### CSS-in-JS
+
+There's many different CSS-in-JS libraries. The two most popular are [styled-components](https://styled-components.com/) and [Emotion](https://emotion.sh/docs/introduction).
+
+There's also zero-runtime solutions like [Linaria](https://github.com/callstack/linaria), where CSS is extracted to CSS files at build-time.
+To see comparisions between different libraries, see [CSS-in-JS benchmarks.](https://github.com/A-gambit/CSS-IN-JS-Benchmarks)
+
+<ProsCard
+  title="CSS-in-JS"
+  pros={[
+    'Easy to distribute as reusable components on NPM.',
+    'You can take advatange of anything in the JavaScript ecosystem.',
+    'Styles are colocated with components, making it easy to delete code.',
+    'Easy to provide multiple themes.'
+  ]}
+/>
+
+<ConsCard
+  title="CSS-in-JS"
+  cons={[
+    'Requires confguring a toolchain like Webpack.',
+    'Some argue it breaks separation of concerns.',
+    'Difficult for libraries, since all consumers must use the same CSS-in-JS solution.'
+  ]}
+/>
+
+### Theme UI / Styled System
+
+Both [Theme UI](https://theme-ui.com) and [styled-system](https://styled-system.com/) abide by the [System UI theme specification](https://system-ui.com/).
+The intent behind this specification is to help make UI components, libraries, and tools as interoperable as possible.
+
+The primary use case for these libraries is implenting a component library and design system.
+One of my favorite libraries is [Chakra UI](https://next.chakra-ui.com/).
+
+<ProsCard
+  title="Theme UI / styled-system"
+  pros={[
+    'Great for component libraries. Many are built on top of this.',
+    'Allows you to easily enforce a design system.',
+    'Spend less time trying to name components',
+    'All libraries that follow System UI spec should be interoperable.'
+  ]}
+/>
+
+<ConsCard
+  title="Theme UI / styled-system"
+  cons={[
+    'You dislike providing styles as props.',
+    'Some argue it breaks separation of concerns.'
+  ]}
+/>
+
+### Tailwind CSS
+
+[Tailwind CSS](https://tailwindcss.com/) has been rapidly growing in popularity. Many people are fed up with the bloat of utility frameworks like
+[Bootstrap](https://getbootstrap.com/) and want an alternative. Tailwind provides an impressive developer experience with a refined API.
+
+With the combination of [Tailwind UI](https://tailwindui.com/), you can easily build extensible, well-designed applications.
+
+<ProsCard
+  title="Tailwind"
+  pros={[
+    'Utility classes allow you to never leave your HTML.',
+    'Allows you to easily enforce a design system.',
+    'Spend less time trying to name components',
+    'All libraries that follow System UI spec should be interoperable.'
+  ]}
+/>
+
+<ConsCard
+  title="Tailwind"
+  cons={[
+    'You can have really, really long class names.',
+    'To use effectively, requires toolchain with PurgeCSS.',
+    'You need to learn Tailwind syntax, which is slighty different than standard CSS.'
+  ]}
+/>
+
```

Comment:
at the end I would add a poll or ask for comments on Twitter (can bump engagement)